### DISCUSSION TITLE
AI-on-the-edge and ESPHome Updates

### DISCUSSION THREAD

**Discussion by JAudi23** [jomjol/AI-on-the-edge-device#2061] [General] [answered]
I am a little confused on whether the ESPHome firmware and the AI-on-the-edge code sit side by side on the device?
I had everything up and running and working nearly perfectly, when I decided to updated the ESP32-Cam device via Home Assistant's ESPHome add-on.
Now for some reason I am unable to access the device by IP.  
Am I missing something, or did this update just cause an issue that I need to reinstall AI-on-the-edge again to recover?

**Comment by JAudi23** [comment]
I think I found my answer with this  linked post.  Can someone confirm for sure that it is ESPHome firmware, OR this AI-on-the-edge firmware to be installed, not both?
https://github.com/jomjol/AI-on-the-edge-device/discussions/657#discussioncomment-2160789
Thank you in advance.

**Comment by jomjol** [comment, marked as answer]
ESPHome is something completly different and cannot run in parallel on the same device side by side

**Comment by JAudi23** [comment]
Thank you, now I know, that clears it up.  I will try to redeploy this AI-on-the-edge to the device.  Hoping I didn't lose my configs.
